tests/test_k8s.py

```diff
@@ -90,7 +93,8 @@ def test_wait_skips_pod_when_status_is_none(self, mock_watch, mock_get_namespace
 
     @patch('calrissian.k8s.watch')
     def test_wait_skips_pod_when_state_is_running(self, mock_watch, mock_get_namespace, mock_client):
-        mock_pod = Mock(status=Mock(container_statuses=[Mock(state=Mock(running=Mock(), terminated=None, waiting=None))]))
+        mock_pod = Mock(
+            status=Mock(container_statuses=[Mock(state=Mock(running=Mock(), terminated=None, waiting=None))]))
```

Comment:
You might be able to simplify creating `mock_pod` into
```
mock_pod = create_autospec(V1Pod)
```
After importing `V1Pod`: `from kubernetes.client.models import V1Pod`
And  create_autospec from mock: `from unittest.mock import create_autospec`